common/types/mapper/thrift/shared.go

```diff
@@ -7936,3 +7936,42 @@ func ToStickyWorkerUnavailableError(t *shared.StickyWorkerUnavailableError) *typ
 		Message: t.Message,
 	}
 }
+
+func FromIsolationGroup(t *types.IsolationGroupConfiguration) *shared.IsolationGroupConfiguration {
```

Comment:
these methods already exist, if you search `isolationGroupConfigToIDL` in this folder